src/lib/y2storage/dialogs/issues.rb

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast2/popup"
+require "y2storage/widgets/issues"
+
+module Y2Storage
+  module Dialogs
+    # Popup to show storage issues
+    #
+    # TODO: The class Yast2::Popup only shows text. But sometimes it is needed to show other kind of
```

Comment:
Please, create a trello card for this

Reply:
https://trello.com/c/RAubMPEA/4990-reporter-and-popup-with-generic-content